File: Seminar/src/main/java/org/sopt/Seminar/domain/category/exception/CategoryNotFoundException.java
package org.sopt.Seminar.domain.category.exception;

import org.sopt.Seminar.global.common.exception.ErrorType.NotFound;
import org.sopt.Seminar.global.common.exception.SoptException;

public class CategoryNotFoundException extends SoptException.NotFoundException {
    public CategoryNotFoundException() {
        super(NotFound.CATEGORY_NOT_FOUND, "해당하는 카테고리가 없습니다.");
    }
}

File: kopring/week2/kotlin-semina/src/main/kotlin/org/sopt/kotlinsemina/domain/category/exception/CategoryNotFoundException.kt
package org.sopt.kotlinsemina.domain.category.exception

import org.sopt.kotlinsemina.global.common.exception.ErrorType.NotFound.*
import org.sopt.kotlinsemina.global.common.exception.SoptException

class CategoryNotFoundException: SoptException.NotFoundException(CATEGORY_NOT_FOUND, "해당하는 카테고리가 없습니다.")